packages/block-library/src/read-progress/block.json

```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "core/read-progress",
+	"category": "theme",
+	"title": "Read Progress",
+	"description": "Displays a read progress. Useful for tracking read progress.",
```

Comment:
"Displays a read progress" isn't quite right, "read" is past tense so I would say instead it displays a "reading progress bar" or a "reading progress indicator".